document-store/src/integrationTest/java/org/hypertrace/core/documentstore/DocStoreQueryV1Test.java

```diff
@@ -1334,6 +1335,24 @@ public void testQueryV1AggregationFilterWithWhereClause(String dataStoreName) th
         dataStoreName, resultDocs, "query/test_aggr_filter_and_where_filter_result.json", 2);
   }
 
+  @ParameterizedTest
+  @ArgumentsSource(AllProvider.class)
```

Comment:
Wondering how this test is passing for Postgres since we didn't add an impl. for the same.! 😕